dataactcore/scripts/loadDUNS.py

```diff
@@ -70,35 +69,35 @@ def activation_check(data, activated_models, benchmarks=False):
         logger.info("Activation check took {} seconds".format(time.time()-activation_check_start))
     return data
 
-
-def update_duns(models, new_data, benchmarks=False):
-    """Modify existing models or create new ones"""
-    logger.info("updating duns")
-    if benchmarks:
-        update_duns_start = time.time()
-    for _, row in new_data.iterrows():
-        awardee_or_recipient_uniqu = row['awardee_or_recipient_uniqu']
-        if awardee_or_recipient_uniqu not in models:
-            models[awardee_or_recipient_uniqu] = DUNS()
-        for field, value in row.items():
-            setattr(models[awardee_or_recipient_uniqu], field, value)
-    if benchmarks:
-        logger.info("Updating duns took {} seconds".format(time.time() - update_duns_start))
+# Removed this function when adding registration_date
```

Comment:
If we removed this function why are we keeping it?

Reply:
We're keeping it because they changed their minds about what should be pulled, so we decided to make sure we keep the previous changes in case this changes once more.